woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Dialog: Refactor Interactivity API actions

This PR changes the Product Gallery Dialog Interactivity API action names and location.



## What

Fixes #11311

## Why

Since the `openDialog` action is shared across components it doesn't make sense for it to live inside of the `Large Image`.

The `openDialog` and `closeDialog` names are a bit more informative about the actual actions performed by the action.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._


1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Ensure the Thumbnails are enabled.
6. Open a product page on the frontend **with multiple product images** (more than `Number of Thumbnails` setting) and test the frontend. Make sure the last thumbnail displays an overlay that opens the Dialog on click.
7. Make sure there are no regressions and the Dialog can still be triggered by clicking the Large Image.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
